src/PAModel/Serializers/SourceSerializer.cs

```diff
@@ -629,6 +639,11 @@ public static void SaveAsSource(CanvasDocument app, string directory2, ErrorCont
                 dir.WriteAllJson("", FileKind.Schema, app._parameterSchema);
             }
 
+            if (app._customPageInputsMetadata != null)
```

Comment:
@milantomic6: It would be nice to add this piece of code alongside other packagesdir writes. I believe that's around line 545 (after pcf controls) in this method.

Reply:
moved!